scripts/builds/react-router-dom.js

```diff
@@ -75,11 +75,22 @@ const webModules = [
     },
     external: ['history', 'prop-types', 'react', 'react-router'],
     plugins: [
-      ignore(['prop-types']),
       babel({
         exclude: /node_modules/,
-        presets: ['@babel/preset-modules', '@babel/preset-react'],
-        plugins: ['babel-plugin-dev-expression']
+        presets: [
+          ['@babel/preset-modules', {
+            // Inference of Function.name is not being relied on for Arrow Functions:
+            loose: true
+          }],
+          ['@babel/preset-react', {
+            // compile spread to Object.assign():
+            useBuiltIns: true
+          }]
```

Comment:
Should these options be used on the production file as well? Or just dev?

Reply:
Both options should be used in all environments:
- JSX's usage of `Object.assign()` is 100% supported in `<script type=module>`
- loose mode for `preset-modules` just disables manual `Function.prototype.name` spoofing for arrow functions, which is already impossible to rely on in minified code.